config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -696,7 +696,8 @@ presubmits:
           - '--node-test-args=--feature-gates=LocalStorageCapacityIsolation=true --container-runtime=remote --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
           - --node-tests=true
           - --provider=gce
-          - --test_args=--nodes=1 --focus="\[Serial\]" --skip="\[Flaky\]|\[Benchmark\]|\[NodeSpecialFeature:.+\]|\[NodeSpecialFeature\]|\[NodeAlphaFeature:.+\]|\[NodeAlphaFeature\]|\[NodeFeature:Eviction\]"
+          #TODO: NodeFeature:DevicePluginProbe is being skipped so it can be readded in 1.23 and avoid backporting changes
```

Comment:
1.23 is already shipped. Do you mean 1.24?

Reply:
Maybe just say:

// NodeFeature:DevicePluginProbe tests are failing and being investigated in https://github.com/kubernetes/kubernetes/issues/106635